Rubberduck.Parsing/VBA/ComponentParseTask.cs

```diff
@@ -181,7 +181,7 @@ private CommonTokenStream RewriteAndPreprocess(CancellationToken cancellationTok
  
             var tokenStreamProvider = new SimpleVBAModuleTokenStreamProvider();
             var tokens = tokenStreamProvider.Tokens(code);
-            _preprocessor.PreprocessTokenStream(_module.Name, tokens, new PreprocessorExceptionErrorListener(_module.ComponentName, ParsePass.CodePanePass), cancellationToken);
+            _preprocessor.PreprocessTokenStream(component.ParentProject, _module.Name, tokens, new PreprocessorExceptionErrorListener(_module.ComponentName, ParsePass.CodePanePass), cancellationToken);
```

Comment:
This seems to leak an `IVBProject`.

Reply:
That's a good catch.  I wrongly assumed, from looking at previous code using ParentProject that this was cached (i.e. owned and disposed by the component).

Looking at the RD codebase, there are many accesses to ParentProject without proper disposal.  Mostly from the TypeLib API, where I made the same error, but a few pre-date that.